src/components/MainLayout/MainPage/ScorePanel/PlayerCard/PlayerCardStyled.tsx

```diff
@@ -1,10 +1,12 @@
 import styled from 'styled-components';
 import { colors } from '../../../../../theme/variables';
+import { Type } from '../Player.types';
 
-export const PlayerCardStyled = styled.div`
+export const PlayerCardStyled = styled.div<{ type: Type }>`
```

Comment:
the same question, type of What? type of flayer, type of card? type of army?

Reply:
Ok. I renamed 'type' to 'playerType'